runtimes/datascience/ubi9-python-3.11/kustomize/base/pod.yaml

```diff
@@ -23,7 +23,4 @@ spec:
       volumeMounts:
         - name: config
           mountPath: /opt/app-root/elyra
-  volumes:
-    - name: config
-      configMap:
-        name: req-config
```

Comment:
Good try, but when I run with this, it gives me

```
The Pod "runtime-pod" is invalid: spec.containers[0].volumeMounts[0].name: Not found: "config"
```

It would appear that you need to delete not just the volume, but also the corresponding volume mount above. I'm gonna try it right now, and if it works, I'll add the fix here.

https://github.com/jiridanek/notebooks/actions/runs/12045121568/job/33583256139#step:29:31